airmozilla/manage/tests/views/test_regions.py

```diff
@@ -0,0 +1,67 @@
+from nose.tools import eq_, ok_
+
+from funfactory.urlresolvers import reverse
+
+from airmozilla.main.models import (
+    Region,
+)
+from .base import ManageTestCase
+
+
+class TestRegions(ManageTestCase):
+    def test_regions(self):
+        """Region management pages return successfully."""
+        response = self.client.get(reverse('manage:regions'))
+        eq_(response.status_code, 200)
+
+        region = Region.objects.create(name='New Region')
+        response = self.client.get(reverse('manage:regions'))
+        eq_(response.status_code, 200)
+        eq_(region.name, 'New Region')
+
+    def test_region_new(self):
+        """Adding new region works correctly."""
+        url = reverse('manage:region_new')
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        response_ok = self.client.post(url, {
+            'name': 'testing',
+        })
+        self.assertRedirects(response_ok, reverse('manage:regions'))
+        region = Region.objects.get(name='testing')
+        response_fail = self.client.post(url)
+        eq_(response_fail.status_code, 200)
+        eq_(region.name, 'testing')
+
+    def test_region_remove(self):
+        """Removing a region works correctly and leaves associated locations
+         with null regions."""
+        region = Region.objects.create(
+            name="Something"
+        )
+        self._delete_test(
+            region,
+            'manage:region_remove',
+            'manage:regions'
+        )
+
+        region = Region.objects.get(id=1)
```

Comment:
You're referring to something in the fixture here. Then I would suggest you fetch it by its name instead. That's easier to read.